backend/README.md

```diff
@@ -53,3 +52,59 @@ From the `backend` directory:
 ```
 yarn deploy
 ```
+
+# Python cloud functions
+
+[Python cloud functions](https://cloud.google.com/functions/docs/quickstart-python) are not
+administered by Firebase, though they are associated
+with the same project specified above, so you will need the same access to administer them.
+
+They live in `backend/python-functions/main.py`.
+
+## Testing
+
+In `python-functions` set up your virtual environment:
+
+```sh
+virtualenv venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
+
+Unlike in Firebase, we don't have tools to emulate these functions locally.
+But we can write tests against them, in `python-functions/test.py`. Run them with:
+
+```sh
+python -m unittest
+```
+
+## Development
+
+While we cannot emulate the Cloud Functions environment locally, we can deploy separate
+development and production versions of the function to facilitate end-to-end development
+and more rigorous pre-release testing.
+
+All developers' local builds of the dashboard will consume the `development` version of
+the function, so please be aware of how any changes you make will affect others and take
+reasonable measures to avoid breaking it and let others know if it will be unavailable
+or unstable for a time while you are actively developing it.
```

Comment:
@jessex note that a subsequent PR to wire up the JS app will have to implement this using an environment variable or some such